recipes/sagelib/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+export CFLAGS="-w $CFLAGS"
+export CXXFLAGS="-w $CXXFLAGS"
+
+export SAGE_FAT_BINARY=yes
+export SAGE_LOCAL=$PREFIX
+ln -s $PREFIX local
+export SAGE_NUM_THREADS=2
+
+# make build builds sagelib and copies some scripts
+# the scripts can be moved to another package and call it sageruntime
+make build
```

Comment:
Why do you need to call a patched make build? Couldn't you just invoke `setup.py` instead?